demo/summer-2016.html

```diff
@@ -211,11 +211,9 @@
 <script src="kitchen-sink/require.js"></script>
 <script>
 require.config({ paths : { "ace" : "../lib/ace" } });
-require(["ace/ace", "ace/ext/language_tools", "ace/ext/katex"], function(ace, langTools, katex) {
+require(["ace/ace", "ace/ext/language_tools", "ace/ext/katex", "ace/ext/papeeria/katex-previewer"], function(ace, langTools, katex, katexPreviewer) {
```

Comment:
it seems that we move `ace/ext/katex` from this require call inside `initKatex` function. Now we load 118kb of minimized katex javascript pretty early.